src/test/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStepTest.java

```diff
@@ -60,6 +62,28 @@ public void invokeReadResolve_nullLogParams_shouldInitiateToDefault() {
         Assert.assertEquals(buildStep.getLogFolder(), SealightsCLIBuildStep.DEFAULT_LOGS_FOLDER);
     }
 
+    @Test
+    public void createCliRunner_commandModeHasLabId_shouldUseIt() {
+        SealightsCLIBuildStep buildStep = new SealightsCLIBuildStep(true, false, mock(CommandMode.class),
+                null, null, null, null, null);
+
+        CommandMode.EndView endView = new CommandMode.EndView(buildSessionId, labId, null);
+        CLIRunner cliRunner = buildStep.createCLIRunner(endView);
+
+        Assert.assertEquals(cliRunner.getLabId(), labId);
+    }
+
+    @Test
+    public void createCliRunner_commandModeHasNoLabId_shouldSetToNull() {
+        SealightsCLIBuildStep buildStep = new SealightsCLIBuildStep(true, false, mock(CommandMode.class),
+                null, null, null, null, null);
+
+        CommandMode.EndView endView = new CommandMode.EndView(buildSessionId, null, null);
```

Comment:
What am I missing?
The two tests are exactly the same besides the fact that one time `labId` is defined and one time you are using `null`.
Am I missing something?

Reply:
Actually what i testing is the NPE i discovered yesterday when you don't have labid at all, no error is thrown